b3lb/rest/models.py

```diff
@@ -408,9 +408,51 @@ def records_effective_hold_time(self):
 class SecretAdmin(ModelAdmin):
     model = Secret
     list_filter = [('tenant', RelatedOnlyFieldListFilter)]
-    list_display = ['__str__', 'description', 'endpoint', 'recording_enabled', 'attendee_limit', 'meeting_limit']
+    list_display = ['__str__', 'description', 'endpoint', 'api_mate', 'recording_enabled', 'attendee_limit', 'meeting_limit', ]
     actions = [records_on, records_off]
 
+    def api_mate(self, obj):
+        low_slug = str(obj.tenant.slug).lower()
+        low_slug_id = f"{low_slug}-{str(obj.sub_id).zfill(3)}"
+        params = {
+            "sharedSecret": obj.secret,
+            "name": f"API Mate test room for {low_slug_id}",
+            "attendeePW": get_random_string(6, '0123456789'),
+            "moderatorPW": get_random_string(6, '0123456789')
+        }
+        slide_string = ""
+        try:
+            slide = Asset.objects.get(tenant__secret=obj).slide
+            if slide:
+                slide_string = f"pre-upload=https://{st.B3LB_API_BASE_DOMAIN}/b3lb/t/{low_slug}/slide"
+        except Asset.DoesNotExist:
+            pass
+
+        if obj.recording_enabled and obj.tenant.recording_enabled:
+            params["record"] = True
+
+        # Todo
+        #   don't send params or get correct way to send "key=value" format via url
+
+        # custom_parameters = ""
+        # for custom_parameter in Parameter.objects.filter(tenant__secret=obj):
+        #     if custom_parameter.mode != Parameter.BLOCK:
+        #         if not custom_parameters:
+        #             custom_parameters = f"{custom_parameter.parameter}={custom_parameter.value}"
+        #         else:
+        #             custom_parameters += f"\n{custom_parameter.parameter}={custom_parameter.value}"
+
+        url = f"https://mconf.github.io/api-mate/?server=https://{obj.endpoint}/bigbluebutton&{urlencode(params)}"
```

Comment:
The api mate URL should be configurable (see also #83) to allow air gaped installations. The default value should be `https://mconf.github.io/api-mate/`.

**You must not use `?`** for the parameters since this leads to get parameter logging (use `#` instead).

Reply:
Used  parameter for base URL and replaced `?` with `#`.